data/translations/ja.ts

```diff
@@ -69,5 +69,49 @@
         <source>Select your user and enter password</source>
         <translation>ユーザ名を選び、パスワードを入力してください</translation>
     </message>
+    <message>
+        <source>Please enter a password!</source>
+        <translation>パスワードを入力してください!</translation>
+    </message>
+    <message>
+        <source>Change password</source>
+        <translation>パスワードを変更</translation>
+    </message>
+    <message>
+        <source>Enter your username</source>
+        <translation>ユーザ名を入力</translation>
+    </message>
+    <message>
+        <source>Enter your password</source>
+        <translation>パスワードを入力</translation>
+    </message>
+    <message>
+        <source>Password:</source>
+        <translation>パスワード:</translation>
+    </message>
+    <message>
+        <source>Show password</source>
+        <translation>パスワードを表示する</translation>
+    </message>
+    <message>
+        <source>Hide password</source>
+        <translation>パスワードを隠す</translation>
+    </message>
+    <message>
+        <source>Suspend</source>
+        <translation>サスペンド</translation>
+    </message>
+    <message>
+        <source>Hibernate</source>
+        <translation>ハイバネート</translation>
+    </message>
+    <message>
+        <source>Password change aborted because maximum tries reached</source>
+        <translation>最大試行回数に達しましたため、パスワード変更は中止しました</translation>
```

Comment:
For messages on the software UI, 「達しましたため」 is a bit too formal.

```suggestion
        <translation>最大試行回数に達したため、パスワード変更は中止しました</translation>
```

Reply:
Is this text used anywhere?
I quickly searched this repository and suspect this text is no longer used.

Reply:
These are from [TextConstants.qml](https://github.com/sddm/sddm/blob/2d27436bab48eb49e4607ab073ab7ad1a98b9350/components/common/TextConstants.qml#L4), and themes use it for common strings. For instance, [sddm-astronaut-theme](https://github.com/search?q=repo%3AKeyitdev%2Fsddm-astronaut-theme%20textconstants&type=code); [other search results](https://github.com/search?q=sddm+theme+textconstants+NOT+language%3Axml&type=code)

Reply:
Got it. The text is used in another repositories.